ChangeLog.md

```diff
@@ -135,6 +137,7 @@
 
   * Change element documentation to add placeholder attribute to `pl-string-input` (Mariana Silva).
 
+  * Change test cases to use templated DB for faster performance (Dave Mussulman).
```

Comment:
Shift ChangeLog entries up to the new version area (this sucks, I know -- I'm trying to think of a better way to handle this).